.eslintrc.js

```diff
@@ -1,10 +1,12 @@
-{
+module.exports = {
 	"root": true,
-	"plugins": ["node"],
+	"plugins": ["node", "prettier"],
```

Comment:
You forgot to add the prettier rule to the eslint rules. This does nothing currently.

https://github.com/not-an-aardvark/eslint-plugin-prettier#usage